rest-service/manager_rest/rest/resources_v3_1/status.py

```diff
@@ -63,14 +63,19 @@
     'cloudify-syncthing.service': 'File Sync Service'
 }
 
-SUPERVISORD_SERVICES = {
-    'AMQP-Postgres': 'cloudify-amqp-postgres',
-    'Cloudify Composer': 'cloudify-composer',
-    'Cloudify Console': 'cloudify-stage',
-    'Management Worker': 'cloudify-mgmtworker',
-    'Webserver': 'cloudify-nginx',
-    'Manager Rest-Service': 'cloudify-restservice',
-    'File Sync Service': 'cloudify-syncthing'
+SUPERVISORD_BASE_SERVICES = {
+    'cloudify-amqp-postgres': 'AMQP-Postgres',
```

Comment:
Aside from the .service extension, these are the same as the above- and I think systemd works without the .service extension so maybe we should remove it and fold them into one?

Reply:
If we can drop the `.service` extension then yeah we can fold them into one instead of having two extra dicts. Will check and see

Reply:
I think this is not going to only drop it from this dict but need also to do changes for the file names as well and for example `nginx.service` is something that we are not configure/handle. the same thing applies for database service name. So maybe we can do it as part of another jira in the future

Reply:
Do we use these to check file names then, or just systemctl status?

Reply:
@geokala we used them when configure each service by moving each service under `/usr/lib/systemd/system/`